src/renderer/src/components/clock.tsx

```diff
@@ -1,26 +1,45 @@
+import useUtilStore from '@renderer/store/utilStore';
```

Comment:
Clock is a nice addition, but is missing from mini-mode. Can it get added to mini-mode somewhere too?

Reply:
Can do, never thought about that, is it needed though since is mini mode not usually used when paired with a client that also has a clock?

Reply:
@pierr3 what do you think?

Reply:
Well, by that logic, it wouldn't be needed with maxi-mode either would it? 😄 TrackAudio is always paired with a client that has a clock.